roles/openshift_logging/README.md

```diff
@@ -12,7 +10,156 @@ generation for Elasticsearch (it uses JKS) as well as openssl to sign certificat
 As part of the installation, it is recommended that you add the Fluentd node selector label
 to the list of persisted [node labels](https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-node-host-labels).
 
-### Required vars:
+### Change to variable structure and deployment structure:
+
+Previously, the EFK stack was deployed as either an all-in-one or a split (apps and ops) deployment based on the boolean variable
+`openshift_logging_use_ops`. This has been phased out in favor of being able to configure multiple (currently up to two is supported)
+ES clusters.
+
+To be able to accomplish this, the old variable naming structure has been revamped to follow a dictionary structure as such:
+
+```yaml
+openshift_logging:
+  cluster_names: ['es']
+  fluentd:
+    nodeselector: {'logging-infra-fluentd': 'true'}
+    cpu_limit: null
+    cpu_request: 100m
+    memory_limit: 512Mi
+    journal_source: ""
+    journal_read_from_head: ""
+    hosts: ['--all']
+    buffer_queue_limit: 32
+    buffer_size_limit: 8m
+    app:
+      host: logging-es
+      port: 9200
+      client_cert: /etc/fluent/keys/cert
+      client_key: /etc/fluent/keys/key
+      ca: /etc/fluent/keys/ca
+
+    infra:
+      host: logging-es
+      port: 9200
+      client_cert: /etc/fluent/keys/infra-cert
+      client_key: /etc/fluent/keys/infra-key
+      ca: /etc/fluent/keys/infra-ca
+
+  mux:
+    allow_external: False
+    use_mux: "{{ openshift_logging_mux_allow_external | default(False) }}"
+    hostname: "{{ 'mux.' ~ openshift_master_default_subdomain}}"
+    port: 24284
+    cpu_limit: null
+    memory_limit: 512Mi
+    cpu_request: 100m
+    # the namespace to use for undefined projects - users will typically not need to set this
+    # extra namespaces to create for mux clients - users will need to set this
+    namespaces: []
+
+  es:
+    install: False
+    master_url: "https://kubernetes.default.svc.{{ openshift.common.dns_domain }}"
+    master_public_url: "{{ 'https://' + openshift_master_cluster_public_hostname | default(openshift.common.public_hostname, true) + ':' ~ openshift_master_api_port }}"
+    nodeselector: null
+    labels: {}
+    purge_logging: False
+    image_pull_secret: ""
+
+    curator:
```

Comment:
I know this is the README but incase people copy/paste it looks like the indentation is off for a few of the components

Reply:
Nope, this is actually as intended. We want to nest Kibana, Curator, and ES under the cluster name, since they are closely tied together in that aspect. Fluentd, Mux, and EventRouter all are at the same level as the cluster names because they can be configured independently of the clustered component. This way we can deploy our aggregator separate from our Storage and UI components.

I may also have an `install` option for each component so someone could do something like deploy without Kibana if desired.